e2e/multi-env-project/multi_env_suite_test.go

```diff
@@ -0,0 +1,45 @@
+package multi_env_app_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/amazon-ecs-cli-v2/e2e/internal/client"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var cli *client.CLI
+var projectName string
+var testEnvironmentProfile string
+var prodEnvironmentProfile string
+
+func TestMultiEnvProject(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Multiple Environments Suite")
+}
+
+var _ = BeforeSuite(func() {
+	testEnvironmentProfile = "e2etestenv"
+	prodEnvironmentProfile = "e2eprodenv"
```

Comment:
How are we eventually going to tackle this in CodeBuild? I guess this question even applies to the "default" profile.

What I can think of:
1. Store the keys and secrets in SSM
2. Inject as env var
3. Run "aws configure" and create these profiles in this `BeforeSuite`

Reply:
I was going to inject them from Secrets Manager / SSM. For local testing, i was going to mount the user's .aws